benchmarks/run_release_benchmark.py

```diff
@@ -0,0 +1,102 @@
+import boto3
+import click
+
+
+def get_boto_client(service_name, region_name):
+    client = boto3.client(
+        service_name,
+        region_name=region_name,
+    )
+    return client
+
+
+def get_boto_resource(service_name):
+    client = boto3.resource(
+        service_name,
+        region_name='us-east-1',
+    )
+    return client
+
+
+def start_benchmark(model_name, model_mode, batch_size, branch, ami, bucket_name, iam_role, region_name,
+                    instance_type='cpu'):
+    benchmark_config_path = f"preset_configs/{model_name}/{instance_type}/{model_mode}_batch_{batch_size}.json"
+
+    s3_path = f"{model_name}/{instance_type}/{model_mode}_batch_{batch_size}"
+
+    user_data = f'''#!/bin/bash -xe
+set -eou pipefail
+
+cleanup()
+{{
+aws s3api put-object --bucket {bucket_name} --key {s3_path}/exec_log.log --body /var/log/user-data.log
+sudo shutdown -h now
+}}
+
+cd ~
+exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
+pip3 install -U awscli
+sudo apt-get install -y apache2-utils
+
+trap 'cleanup;exit 1' SIGINT SIGTERM EXIT
+git clone https://github.com/pytorch/serve.git
```

Comment:
This is great ! 

Can you consider moving the bash script text to a separate file a large bash script text inside a python script seems very cryptic

Reply:
Done. Moved the user data to a template file, now rendered using jinja2.